basic/.travis.yml

```diff
@@ -1,8 +1,7 @@
 language: node_js
 
 node_js:
-  # Node 10.3+ includes npm@6 which has good "npm ci" command
-  - 10.8
+  - 18
```

Comment:
Cypress v12 dropped node 12 support. This is an example for users. Updated to use the latest LTS node version.